spec/system/placements/schools/school_contacts/user_adds_a_placement_contact_spec.rb

```diff
@@ -0,0 +1,144 @@
+require "rails_helper"
+
+RSpec.describe "Adding a placement contact to a school", service: :placements, type: :system do
+  let(:school) { build(:placements_school, with_school_contact: false) }
```

Comment:
As discussed in Slack, can we please try removing `let` blocks in favour of instance variables?

So move this to the `given_i_am_signed_in` step as:

```ruby
def given_i_am_signed_in
  @school = build(:placements_school, with_school_contact: false)
  sign_in_placements_user(organisations: [@school])
end
```

and then other steps which need access to the school can now use `@school`.